R/site_details_metric_meta_data_list.R

```diff
@@ -0,0 +1,34 @@
+#' Creates taglist to summarize participants at a particular site
+#' @inheritParams shared-params
+#' @export
+#' @keywords internal
+site_details_metric_meta_data_list <- function(dfMetrics, rctv_strMetricID) {
```

Comment:
```suggestion
site_details_metric_meta_data_list <- function(dfMetrics, strMetricID) {
```
Since we're evaluating `rct_strMetricID` before passing it into here, this is no longer reactive. We should also add `strMetricID` to `shared-params`.